pkg/core/storage/store.go

```diff
@@ -45,6 +45,21 @@ const (
 	MaxStorageValueLen = 65535
 )
 
+// SeekRange represents options for Store.Seek operation. Prefix denotes the Seek's
+// lookup key; empty/nil Prefix means seeking through all keys in the DB starting from
+// the Start if specified. Start denotes the key to start Seek from (including the
+// key itself); if no matching key was found then next suitable key is picked up;
+// Start must be either empty or be prefixed by Prefix; empty/nil Start means seeking
+// starting from Prefix if specified. Empty/nil Prefix and empty/nil Start can be combined.
+// Backwards denotes whether Seek direction should be reversed, i.e. whether seeking
+// should be performed in a descending way; Backwards can be combined with Prefix and
+// Start.
```

Comment:
Can we make it more readable by describing each case on a new line?
`Empty/nil` can be replaced with `Empty` without hurting readability.
Also `empty/nil Start means seeking starting from prefix` to me looks like prefix is a replacement for start which is not the case. I think documenting individual fields is the way to go here.

Reply:
Fixed, see the updated comment.